### ISSUE TITLE
BRAT Summary Report

### ISSUE THREAD

**Issue by MatthewMeier** [Riverscapes/pyBRAT#46]
Looking into the output for the BRAT Summary Report tool the attribute table fields that are with the dam point shapefile are being added to the output from this tool. Due to multiple dams being snapped to some segments these point shapefile fields being carried over are not applicable and should not be included in the output. @banderson1618 @wally-mac

**Comment by banderson1618** [comment]
What are the names of the fields that need to be deleted? That should be a pretty easy fix.

**Comment by MatthewMeier** [comment]
it is whatever extra fields are in the point shapefile so it changes with every file. If this is not an easy fix if they are all different we can do it manually?

**Comment by banderson1618** [comment]
No, I can fix that. I'll just delete any fields that aren't in the input and aren't created by the tool.
